client/my-sites/theme/style.scss

```diff
@@ -383,21 +382,33 @@ $theme-sheet-content-max-width: 1462px;
 }
 
 .theme__sheet-web-preview {
+	$iframe-viewport-height: 1040px;
+	$iframe-viewport-height-scaled: calc($iframe-viewport-height / 2);
+
+	min-height: $iframe-viewport-height-scaled;
 	pointer-events: none;
 
 	@include breakpoint-deprecated( ">960px" ) {
-		height: calc(100vh - 64px);
 		pointer-events: all;
 		position: sticky;
 		top: 16px;
 
-		.theme-preview__frame-wrapper .theme-preview__frame {
-			height: 200%;
-			max-width: 200%;
-			transform: scale(0.5) !important;
-			width: 200%;
+		.theme-preview__frame-wrapper {
+			max-height: 100vh;
```

Comment:
Can you try `max-height: calc( 100vh - 32px - var(--masterbar-height));` to rest the `16px` top/bottom paddings and the header height?

It seems to work:

<img width="1426" alt="Screenshot 2566-07-06 at 16 31 43" src="https://github.com/Automattic/wp-calypso/assets/1881481/213445e3-5d6c-4240-8851-ad338f221a4b">

I found the variable here:

<img width="315" alt="Screenshot 2566-07-06 at 16 28 54" src="https://github.com/Automattic/wp-calypso/assets/1881481/d3eec80f-598f-46a0-8f60-9b9b1fa49e8f">

Reply:
Tested your suggestion, and it looks great!

Reply:
We might need another value for the LoTS but it's a minor issue

![image](https://github.com/Automattic/wp-calypso/assets/13596067/180256ad-43fa-497c-9b3e-8e3761108f02)